test/cookie.test.js

```diff
@@ -942,3 +942,311 @@ test('if cookies are not set, then the handler creates an empty req.cookies obje
     t.equal(res.statusCode, 200)
   })
 })
+
+test('issue 208 - logging enabled - below', (t) => {
```

Comment:
Please change the description. It is not clear from just reading the test description what you actually test

atleast put the name of the flag into the description
```suggestion
test('enableWarnOnSafeLimit - logging enabled - serialized cookie is smaller than the safe limit', (t) => {
```